lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -215,45 +202,3 @@ function dosomething_northstar_build_ns_user($user, $form_state) {
   // Return fully built northstar user object.
   return $ns_user;
 }
-
-
-class Northstar {
```

Comment:
if we can autoload we can switch to guzzle and update to the code that was commented out :)